<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslationWidget</class>
 <widget class="QWidget" name="TranslationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
	background-color:rgb(250,250,250);
	border-radius:10px;
}
QComboBox{
	border:1px solid rgb(200,200,200);
	border-top-left-radius: 5px;
	border-top-right-radius:5px;
	background-color: rgb(255, 255, 255);
	color:rgb(0,0,0);
}
QComboBox::drop-down{
	border:0px;
}
QComboBox QAbstractItemView{
	border:none;
	background-color:rgb(255, 255, 255);
	color:rgb(0,0,0);
	selection-background-color: rgb(200, 200, 200);
	selection-color:rgb(255, 255, 255);
}
QComboBox QAbstractItemView:!enabled{
    color:rgb(125,125,125);
}
QPushButton{
	border-radius:5px;
	color: rgb(0,0, 0);
	background-color: rgb(255, 255, 225);
}
QPushButton:hover{
	border-radius:5px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 252);
}
QTextEdit{
	border-radius:5px;
	border:1px solid rgb(200,200,200);
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}
QScrollBar:vertical{
	border: none;
	width:8px;
	margin: 0 0 0 0;
	background: transparent;
}
QScrollBar::handle:vertical{
	background: rgb(200, 200, 200);
	border-radius:4px;
}
QScrollBar::handle:vertical:hover{
	background: rgb(160, 160, 160);
	border-radius:4px;
}
QScrollBar::handle:vertical:pressed{
	background: rgb(5, 160, 255);
	border-radius:4px;
}
QScrollBar::add-page,QScrollBar::sub-page:vertical{
	background: transparent;
}
QScrollBar::add-line,QScrollBar::sub-line:vertical{
	background: transparent;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="addButton">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>添加译文</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QTextEdit" name="textEdit">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="typeCcomboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>有道翻译</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>百度翻译</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="dirCcomboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>英转中</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中转英</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
